pkgs/by-name/re/recon-ng/package.nix

```diff
@@ -0,0 +1,71 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+  substituteAll,
+  fetchpatch,
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "recon-ng";
+  version = "5.1.2";
+
```

Comment:
```suggestion
  pyproject = true;

```

Reply:
Adding this actually breaks the build of an otherwise working app. Not sure what to do about that.

Reply:
See my latest comment.